ml/ml.cc

```diff
@@ -436,6 +436,10 @@ const char *db_models_delete =
     "DELETE FROM models "
     "WHERE dim_id = @dim_id AND before < @before;";
 
+const char *db_models_prune =
+    "DELETE FROM models "
+    "WHERE after < @after LIMIT @n;";
```

Comment:
You can use `WHERE after < UNIXEPOCH() - @after` and bind `@after` directly to `Cfg.delete_models_older_than` to avoid the cast to int

Reply:
I don't want to overload the meaning of `after`, do you mind keeping the extra cast?

Reply:
Ok lets leave it for now